samples/nextjs/scripts/generate-config.ts

```diff
@@ -49,7 +47,8 @@ const config = {};\n`;
   });
   // Set computed values
   Object.keys(computedConfig).forEach((prop) => {
-    configText += `config.${prop} = ${computedConfig[prop]};\n`;
+    // eslint-disable-next-line prettier/prettier
```

Comment:
Why are we disabling prettier here?

Reply:
My brain just didn't want to handle the difference with line breaks between non-computed and computed values 😄 i.e.

```
  // Set base configuration values, allowing override with environment variables
  Object.keys(config).forEach((prop) => {
    configText += `config.${prop} = process.env.${constantCase(prop)} || "${config[prop]}",\n`;
  });
  // Set computed values
  Object.keys(computedConfig).forEach((prop) => {
    configText += `config.${prop} = process.env.${constantCase(prop)} || ${
      computedConfig[prop]
    };\n`;
  });
```

I'll remove the prettier comment.